js/start.md

```diff
@@ -851,56 +913,120 @@ new Vue({
 }).$mount('#app')
 ```
 
-Next, open `src/components/HelloWorld.vue`.
-
-Add the following to the `<template>` element: 
+Next, open `App.vue` add data to your database with a mutation by using `API.graphql()`:
 
 ```html
-  <div class="hello">
-    <h1>{{ msg }}</h1>
-    <p>
-      For a guide and recipes on how to configure / customize this project,<br>
-      check out the
-      <a href="https://cli.vuejs.org" target="_blank" rel="noopener">vue-cli documentation</a>.
-    </p>
-    <button v-on:click="handleAnalyticsClick">Generate Analytics Event</button>
-    <div v-if="analyticsEventSent">
-      <p>Event Submitted.</p>
-      <p>Events sent: {{ eventsSent }}</p>
-      <a v-bind:href="url" target="_blank">View Events on the Amazon Pinpoint Console</a>
-    </div>
+<template>
+  <div id="app">
+    <button @click="createNewTodo">Add Todo</button>
   </div>
+</template>
+
+<script>
+import API, {  graphqlOperation } from '@aws-amplify/api';
+// eslint-disable-next-line
+import PubSub from '@aws-amplify/pubsub';
+import { createTodo } from "./graphql/mutations";
+
+export default {
+  name: 'app',
+  methods :{
+    async createNewTodo(){
+      const todo = { name: "Use AppSync" , description: "Realtime and Offline"}
+      await API.graphql(graphqlOperation(createTodo, { input: todo }))
+    }
+  }
+};
+</script>
 ```
 
-Add the following to the `<script>` element:
+To display the data, update `App.vue` to list all the items in the database by importing `listTodos` and then using the `created()` Vue lifecycle method to update the page when a query runs on page load:
 
-```js
-import awsconfig from '../aws-exports';
+```html
+<template>
+  <div id="app">
+    <button @click="createNewTodo">Add Todo</button>
+    <ul>
+      <li v-for="todo in todos" :key="todo.id">
+        {{ todo.name }} - {{ todo.description }}
```

Comment:
Must be some sort of md issue here.  When I view the site, I see:
```
<template>
  <div id="app">
    <button @click="createNewTodo">Add Todo</button>
    <ul>
      <li v-for="todo in todos" :key="todo.id">
         - 
      </li>
    </ul>
  </div>
</template>
```
This generates only a dash for each entry.